### PR TITLE
Entity extraction improvements

### PR DESCRIPTION
**Pull request by fpingham**
Added fix to avoid irrelevant attributes being returned plus an example of extracting unrelated entities and an exampe of using an 'extra_info' attribute to extract unstructured data for an entity.

### CODE REVIEW

**Comment by fpingham** [comment]
Hey @hwchase17, I just rebased this branch to avoid conflicts with master - do you think we can merge? 

People have been asking me about the updated notebooks and I'd rather refer them directly to langchain docs than a branch. Let me know if you think I should make any changes!

File: langchain/chains/openai_functions/extraction.py [hunk 1]
```diff
@@ -32,11 +32,16 @@ def _get_extraction_function(entity_schema: dict) -> dict:
 
 
 _EXTRACTION_TEMPLATE = """Extract and save the relevant entities mentioned\
- in the following passage together with their properties.
+
```
Comment (baskaryan) [inline langchain/chains/openai_functions/extraction.py:35]: don't think we want a new line here, do we?

Comment (fpingham) [reply langchain/chains/openai_functions/extraction.py]: Yeah, that new line was not necessary

**Comment by baskaryan** [comment]
thanks @fpingham!

**Pull request merged**